docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
```

Comment:
As you already added a version at your Dockerfile, you should use an specific version of postgrees here to avoid  issues if a image major release is launched.

Since this file is used only to developmen, you can specify the version `latest` if you don't mind to test the latest version (eventually you'll have to deal with some errors), .

Anyway, it is a good practice specify the version of the image.

Reply:
makes total sense. I will lock the version and we can manually upgrade when we want.